javascript-and-json/main.js

```diff
@@ -0,0 +1,19 @@
+const array = [
+  { title: 'Charolettes Web', author: 'E.B.White', isbn: '1234' },
+  { title: 'Harry Potter', author: 'J.K. Rowling', isbn: '5678' },
+  { title: 'Tidying', author: 'Marie Condo', isbn: '9012' }
+];
+
+console.log('arry:', array);
+console.log('array typeof:,', typeof array);
+
+const jsString = JSON.stringify(array);
+console.log('JSON arry:', JSON.stringify(array));
```

Comment:
You should just be console logging your `jsString` variable. You already ran `JSON.stringify` so just use the variable rather than doing it again